doc/user/UpdaterConfig.md

```diff
@@ -353,15 +353,11 @@ For this to be possible three things need to be configured:
 **Path:** /updaters/[1]   
 **Enum values:** `stations` | `vehicles` | `all`
 
-Temporary parameter. The type of rental data to include.
+Temporary parameter. Use this to specify the type of rental data that is allowed to be read from the data source.
 
-This parameter is temporary and will be removed in a future version of OTP.
```

Comment:
This should still be included in some way, probably replace the "Temporary parameter" in the sentence above this with this content..